en/mkdocs.yml

```diff
@@ -731,6 +731,7 @@ nav:
                         - Configuring Microgateway: observe/api-manager-analytics/configure-analytics/configure-microgateway.md
                         - Configuring API Gateway: observe/api-manager-analytics/configure-analytics/configure-synapse-gateway.md
                     - Working with Organizations: observe/api-manager-analytics/configure-analytics/working-with-organizations.md
+                    - Working with Alerts: observe/api-manager-analytics/configure-analytics/configure-alerts.md
```

Comment:
Shouldn't we name this page 'Configuring Alerts'?

Reply:
I aligned the name with organizations page. isn't it ok to say working with?

Reply:
'Working with' is something we would avoid unless we have to...

Isn't it correct to say 'Configuring'?